starlette_admin/templates/forms/interval.html

```diff
@@ -0,0 +1,33 @@
+<div class="{% if error %}{{ field.error_class }}{% endif %}">
```

Comment:
I would prefer a view like Dimension in this form https://starlette-admin-demo.jowilf.com/admin/mongoengine/product/create. You can copy form https://github.com/jowilf/starlette-admin/blob/main/starlette_admin/templates/forms/collection.html

Reply:
Hey, I'm not sure what to do with these converters..what do they do? I just mimic'd the "Time" one:

```python
    @converts("Interval")
    def conv_interval(self, *args: Any, **kwargs: Any) -> BaseField:
        return IntervalField(
            **self._field_common(*args, **kwargs),
        )
```

Is that it?

Reply:
Yes, that's it. The converter's role is to automatically find the appropriate field for each attribute. For SQLAlchemy, it uses the attribute type. Think of it as a map between the attribute type and the corresponding field.

Reply:
Updated the pr.